We could try 

```
jdk: openjdk8 
```

or fix the error being generated

```
[ERROR] /home/travis/build/TheTransitClock/transitime/transitclockWebapp/src/main/java/org/transitclock/reports/ScheduleAdherenceController.java:[118,103] error: incompatible types: bad type in conditional expression
[ERROR]     Integer cannot be converted to Double
[ERROR] /home/travis/build/TheTransitClock/transitime/transitclockWebapp/src/main/java/org/transitclock/reports/ScheduleAdherenceController.java:[119,100] error: incompatible types: bad type in conditional expression
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,5 +1,5 @@
 language: java
-jdk: oraclejdk8
+jdk: openjdk8
 cache:
   directories:
   - "$HOME/.m2"

